### PR TITLE
Finder favorite position and fixes

### PR DESCRIPTION
**Pull request by gabriel**
Fix for Finder favorite not always getting removed (if keybase directory didn't exist). Saves Finder favorite position (instead of always adding first). Also some go code removal.

Installer changes:
- Saves position of Finder favorite and restores
- Doesn't add Finder favorite if `finder_disabled.config` file present
- Finder favorite removed even if directory not present (fix)
- Don't add Finder favorite on 10.10 because there are Finder favorite bugs causing duplicate entries and other weirdness

Go changes:
- `UninstallKBFSOnStop` calls `uninstallMountDir` separately (fixes Finder favorite not getting removed if directory didn't exist)
- Add `--debug` to native installer calls for more logging
- Remove unused go code `AfterUpdateApply` and `checkFuseUpgrade`, from an older time

### CODE REVIEW

**Comment by gabriel** [comment]
@keybase/react-hackers

File: osx/KBKit/KBKit/System/KBSharedFileList.m [hunk 1]
```diff
@@ -22,7 +22,6 @@ + (NSArray *)itemsForType:(CFStringRef)type {
   UInt32 seed = 0U;
   CFArrayRef itemsRef = LSSharedFileListCopySnapshot(fileListRef, &seed);
   NSArray *items = CFBridgingRelease(itemsRef);
-  CFRelease(itemsRef);
```
Comment (chrisnojima) [inline osx/KBKit/KBKit/System/KBSharedFileList.m]: why did this go away?

**Comment by chrisnojima** [comment]
@maxtaco  can someone from core look at the go code

File: osx/KBKit/KBKit/System/KBSharedFileList.m (hunk 1 above)
Comment (gabriel) [inline osx/KBKit/KBKit/System/KBSharedFileList.m]: It was an over-release that the static analyzer caught.
Above this is CFBridgingRelease which transfers it to ARC.

**Pull request merged**